.gitignore

```diff
@@ -3,11 +3,13 @@
 .DS_Store
 bot.log
 keep-alive.sh
+/.idea
 
 # Secrests
 /secrets.json
 /secrets.txt
 /secrets_5XB.json
+/secrets_W5AC.json
 /secrests_W5AC.json
```

Comment:
Not sure if this is a misspelling in .gitignore of if this is how you have the file @IanDuncanT

Reply:
Nope. Just a misspelling.